pkg/tsdb/cloudmonitoring/types.go

```diff
@@ -46,6 +68,7 @@ type (
 		QueryType    string
 		MetricQuery  metricQuery
 		SloQuery     sloQuery
+		MqlQuery     cloudMonitoringMqlQuery
```

Comment:
[Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
```suggestion
		MQLQuery     cloudMonitoringMqlQuery
```